gems/bootstrap-sass/CVE-2019-8331.yml

```diff
@@ -0,0 +1,20 @@
+---
+gem: bootstrap-sass
+cve: 2019-8331
+url: https://github.com/twbs/bootstrap-sass/releases/tag/v3.4.1
```

Comment:
https://blog.getbootstrap.com/2019/02/13/bootstrap-4-3-1-and-3-4-1/ seems to give more details.

Reply:
Makes sense. I wasn't sure it was clear enough that issues with the larger Bootstrap project will apply to the `bootstrap-sass` gem, so I chose the link that showed the explicit connection first, and added the details to `related`. I'll swap them around and push changes in a moment.

Reply:
Done